src/Torch.Core/Commands/ConsoleService.cs

```diff
@@ -145,7 +205,7 @@ private void RewriteUserInput()
             ClearUserInput();
             var (left, top) = GetUserCursor();
             Console.SetCursorPosition(0, top);
-            _stdOut!.Write($"> {_currentUserInput}");
+            _stdOut!.Write($"> \u001b[33;1m{_currentUserInput}\u001b[0m");
```

Comment:
The ANSI escape codes aren't working in PowerShell, Command Prompt, or Rider's debug console. This should be fixed or we should pass on colored output for now.

Reply:
I believe this requires a P/Invoke call to https://docs.microsoft.com/en-us/windows/console/setconsolemode that sets the `ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004` flag.